### PR TITLE
Bug 1137882 — Log the accurate number of users.

### PR DESCRIPTION
**Pull request by Natim**
https://bugzilla.mozilla.org/show_bug.cgi?id=1137882

### CODE REVIEW

File: loop/middlewares.js [hunk 1]
```diff
@@ -92,6 +93,10 @@ function logMetrics(req, res, next) {
         }
```
Comment (leplatrem) [inline loop/middlewares.js]: Shouldn't we remove the previous one ?

**Comment by leplatrem** [comment]
Could you please add a test asserting the behaviour you're fixing here ?

File: loop/middlewares.js (hunk 1 above)
Comment (Natim) [inline loop/middlewares.js]: At this point the idea was to use the ``participants.length`` in case ``roomParticipantsCount`` is not present.

**Comment by Natim** [comment]
Yes sure.

**Comment by leplatrem** [comment]
r+

**Pull request closed without merging**